tests/spec/core/issues-notes-spec.js

```diff
@@ -43,36 +50,37 @@ describe("Core — Issues and Notes", () => {
         "<p class='note' title='NOT-TIT'>NOTE</p></section>",
     };
     const doc = await makeRSDoc(ops);
-    const $iss = $("div.issue", doc).first();
-    const $atr = $("div.atrisk", doc);
-    const $piss = $iss.find("p");
-    const $patr = $atr.find("p");
-    const $spiss = $("span.issue", doc);
-    const $spatr = $("span.atrisk", doc);
-    const $not = $("div.note", doc);
-    const $pnot = $not.find("p");
-    const $spnot = $("span.note", doc);
+    const iss = doc.querySelector("div.issue", doc);
+    const atr = doc.querySelector("div.atrisk");
+    const spiss = doc.querySelector("span.issue");
+    const spatr = doc.querySelector("span.atrisk");
+    const not = doc.querySelector("div.note");
+    const spnot = doc.querySelector("span.note");
 
-    expect($spiss.parent("div").length).toEqual(0);
-    expect($spatr.parent("div").length).toEqual(0);
-    expect($spnot.parent("div").length).toEqual(0);
+    expect(utils.parents(spiss, "div").length).toEqual(0);
```

Comment:
Comment above applies to these checks.
```suggestion
    expect(spiss.closest("div")).toBeNull();
```
Same with the next two below...